BowOpenAPI/BowOpenAPICommand.swift

```diff
@@ -0,0 +1,17 @@
+//  Copyright © 2019 The Bow Authors.
+
+import ArgumentParser
+
+struct BowOpenAPICommand: ParsableCommand {
+    static var configuration = CommandConfiguration(commandName: "bow-openapi",
+                                                    abstract: "Generate a Swift network client from an OpenAPI / Swagger specification file." )
+    
+    @Option(help: "The name for the output module.")
+    var name: String
+    
+    @Option(help: "The path to schema open api. ex. `/home/schema-openapi.json`.")
```

Comment:
```suggestion
    @Argument(help: ArgumentHelp("The path to schema open api. ex. `/home/schema-openapi.json`.", valueName: "<json | yml>")
    )
```

Reply:
yep, you're right! let me change it 👏

Reply:
updated! :)

Now the help message looks like:

`bow-openapi --help`
```
OVERVIEW: Generate a Swift network client from an OpenAPI / Swagger
specification file.

USAGE: bow-openapi --name <name> --schema <json | yaml> --output <output>

OPTIONS:
  --name <name>           Name for the output module. 
  --schema <json | yaml>  Path to the OpenAPI/Swagger schema. ex.
                          `/home/schema-openapi.json`. 
  --output <output>       Path where the Swift package containing the network
                          client will be generated. ex. `/home`. 
  -h, --help              Show help information.
```